### PR TITLE
Respect prefers-reduced-motion

### LINKED ISSUE #531: Respect prefers-reduced-motion for the typewriter effect on the homepage

### PR DESCRIPTION
**Pull request by davenewham**
Closes #531

### CODE REVIEW

File: src/home.js [hunk 1]
```diff
@@ -39,5 +39,11 @@ function startTextAnimation(i) {
 }
 
 window.onload = () => {
-    startTextAnimation(0);
+    const reducedMotion = window.matchMedia('(prefers-reduced-motion: reduce');
```
Comment (baltpeter) [inline src/home.js:42]:
```suggestion
    const reducedMotion = window.matchMedia('(prefers-reduced-motion: reduce)');
```

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thanks, @davenewham!

There is currently (sometimes) a short flash of the first word before the JS kicks in:

ezgif-7-9b71bdfca345

Not sure what the best way to go about this is. Maybe we should just start off with no word in the HTML?

https://github.com/datenanfragen/website/blob/25b3fa755acce08764d6c0075e893583616e96da/layouts/index.html#L6

**Comment by davenewham** [comment]
Good spot, I didn't notice that. Used a <br> tag to prevent the elements moving around (not sure if there's a better way), however because of that, people viewing with js turned off they will see an extra line between the button and the text.
image

**Review by baltpeter — APPROVED** [review]

**Comment by baltpeter** [comment]
> however because of that, people viewing with js turned off will see an extra line between the button and the text.

I think that's fine. Thanks!

**Pull request merged**